pkg/kube/fake/scripts/resources_generator

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
```

Comment:
How do you start a test cluster for each Kubernetes version? Can't see it anywhere in this PR.

Reply:
You can find description [here](https://github.com/flant/shell-operator/blob/fake-cluster-rework/pkg/kube/fake/resources.go#L6)

Shortly: It's a one time operation. We can use any cluster to take resources manually

Reply:
Please, create an issue. We should have a way to get this from a clean `kind` cluster. That should be deployed via a script.